integrations/nifi/nifi-nar-bundles/nifi-core-bundle/nifi-core-processors/src/main/java/com/thinkbiganalytics/util/ColumnSpec.java

```diff
@@ -107,8 +124,11 @@ public static String toPrimaryKeyJoinSQL(ColumnSpec[] specs, String leftTableAli
                     boolean pk = false;
                     boolean modifiedDt = false;
                     boolean createDt = false;
+                    String otherName = "";
                     switch (len) {
                         default:
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move this default to the end of the switch. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)